xml/Microsoft.Extensions.Primitives/StringSegmentComparer.xml

```diff
@@ -61,10 +61,19 @@
         <Parameter Name="y" Type="Microsoft.Extensions.Primitives.StringSegment" Index="1" FrameworkAlternate="dotnet-plat-ext-2.2;dotnet-plat-ext-3.0;dotnet-plat-ext-3.1;dotnet-plat-ext-5.0" />
       </Parameters>
       <Docs>
-        <param name="x">To be added.</param>
-        <param name="y">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="x">The first object to compare.</param>
+        <param name="y">The second object to compare.</param>
+        <summary>Compares two objects and returns a value indicating whether one is less than, equal to, or greater than the other.</summary>
+        <returns>A signed integer that indicates the relative values of <paramref name="x" /> and <paramref name="y" />, as shown in the following table.   
+ <list type="table"><listheader><term> Value   
+ </term><description> Meaning   
+ </description></listheader><item><term> Less than zero   
+ </term><description><paramref name="x" /> is less than <paramref name="y" />.   
+ </description></item><item><term> Zero   
+ </term><description><paramref name="x" /> equals <paramref name="y" />.   
+ </description></item><item><term> Greater than zero   
+ </term><description><paramref name="x" /> is greater than <paramref name="y" />.   
+ </description></item></list></returns>
         <remarks>To be added.</remarks>
```

Comment:
@maryamariyan the table looks good:
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.primitives.stringsegmentcomparer.compare?branch=pr-en-us-4105&view=dotnet-plat-ext-3.1

But I just noticed it is in the returns section, and it could potentially look bad in IntelliSense. We should instead put the table in the remarks. So here's my suggested change:

```suggestion
        <returns>A signed integer that indicates the relative values of <paramref name="x" /> and <paramref name="y" />.</returns>
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

The value returned by this method can have the following meanings:

| Value             | Meaning                  |
|-------------------|--------------------------|
| Less than zero    | `x` is less than `y`.    |
| Zero              | `x` equals `y`.          |
| Greater than zero | `x` is greater than `y`. |

          ]]></format>
        </remarks>
```